tests/Unit/Domain.LinnApps.Tests/TpkServiceTests/WhenUnpickingStock.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Linn.Stores.Domain.LinnApps.Tests.TpkServiceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+
+    using FluentAssertions;
+
+    using Linn.Stores.Domain.LinnApps.Models;
+    using Linn.Stores.Domain.LinnApps.Requisitions;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenUnpickingStock : ContextBase
+    {
+        private ProcessResult result;
+
+        private IQueryable<ReqMove> moves = new List<ReqMove>
+                                                        {
+                                                            new ReqMove { Sequence = 1, Quantity = 1m, DateCancelled = null, StockLocatorId = 1 },
+                                                            new ReqMove { Sequence = 2, Quantity = 1m, DateCancelled = null, StockLocatorId = 1 },
+                                                            new ReqMove { Sequence = 3, Quantity = 1m, DateCancelled = null, StockLocatorId = 1 }
+                                                        }.AsQueryable();
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.ReqMovesRepository.FilterBy(Arg.Any<Expression<Func<ReqMove, bool>>>())
+                .Returns(this.moves.AsQueryable());
+
+            this.StoresPack.UnpickStock(1, 1, Arg.Any<int>(), 1, 1, 1m, 1, 1)
+                .Returns(new ProcessResult(true, string.Empty));
+
+            this.result = this.Sut.UnpickStock(1, 1, 1, 1, 1, 1, 1);
```

Comment:
could maybe use different numbers just to check the right thing's passed in the right place

Reply:
Its just buttons that let them remove one item or all items from a consignment from the TPK screen. 
For some reason I had assumed that the buttons opened a different form and had missed them out.
Not sure it'd be much use anywhere else